src/characters/zombie.js

```diff
@@ -51,7 +52,7 @@ class Zombie extends Character {
     let coordinates = displayCoordinates(sketch, board, this.x, this.y)
 
     let changeFactor = this.size * coordinates.perspective / img.width
-    sketch.image(img, coordinates.x - (this.size * coordinates.perspective / 2), coordinates.y - (img.height * changeFactor), img.width * changeFactor, img.height * changeFactor)
+    sketch.image(img, coordinates.x - (this.size * coordinates.perspective / 2), coordinates.y - (img.height * changeFactor * (1 - this.shadowSize)), img.width * changeFactor, img.height * changeFactor)
```

Comment:
Please don't do any calculations in parameters sent into the function. Makes for much less readable code.